### DISCUSSION TITLE
Create a FOB using a building existing on map

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1455] [Question and answer] [answered]
Hi,

I would like to Dev a new H&M mission. The first pop will be on a carrier. But i would like to prepare a FOB for the logistic point etc ...
and affect him a clear area.

I would like to create a FOB from an existing building on a map using the ModuleEditTerrainObject_F module in Eden.
I guess i need to put the NATO flag too for the teleportation.

Is it possible and how can i do this ?

Or may be there is an easier way to do it ?

Thanks for your help Vdauphin

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Is it possible and how can i do this ?

No

> Or may be there is an easier way to do it ?

Just create a FOB where is your logistic point? May be lower the`btc_fob_minDistance`
https://github.com/Vdauphin/HeartsAndMinds/blob/91fae918b62e061ea720c01770d8debc498184f1/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L395

Cheers
